src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/MemoryMarshal.cs

```diff
@@ -219,7 +219,8 @@ public static ReadOnlySpan<byte> AsBytes<T>(ReadOnlySpan<T> span)
         /// <returns>A span representing the specified reference and length.</returns>
         /// <remarks>The lifetime of the returned span will not be validated for safety by span-aware languages.</remarks>
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static Span<T> CreateSpan<T>([LifetimeAnnotation(true, false)] ref T reference, int length) => new Span<T>(ref reference, length);
+        public static Span<T> CreateSpan<T>(scoped ref T reference, int length) =>
```

Comment:
Presumably `scoped` is going to find its way into documentation / IntelliSense / etc.?  We should update the XML docs and docs.microsoft.com to make it very clear these APIs are cheating: they're scoped to satisfy the compiler, but they do actually store/return the reference.

Reply:
I'll update the `///` when the official doc is signed off - https://github.com/dotnet/dotnet-api-docs/pull/8187.